browser/extensions/BUILD.gn

```diff
@@ -92,20 +92,20 @@ source_set("extensions") {
     deps += [ "//brave/components/sidebar" ]
   }
 
-  if (brave_wallet_enabled) {
+  if (ethereum_remote_client_enabled) {
     sources += [
       "api/brave_wallet_api.cc",
```

Comment:
this is a bit confusing, maybe this file should be renamed to brave_ethereum_remote_client_api?

Reply:
I'll add a comment but it's because the `ethereum_remote_client` extension uses chrome.braveWallet already so I can't rename the API.